extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosImporter.java

```diff
@@ -214,7 +214,7 @@ private long importPhotoBatch(
 
         try (InputStream s = streamWrapper.getStream()) {
           String uploadToken = getOrCreatePhotosInterface(jobId, authData).uploadPhotoContent(s,
-              photo.getSha1());
+              photo.getSha1(), photo.getTitle());
```

Comment:
note I think this now lives in GPhotosUpload class, so you'll have to revert this file, merge ,and then re-add this param in the new class.